episodes/chapter6.md

```diff
@@ -0,0 +1,570 @@
+  version             Returns current version of a vantage6 node.
+```
+
+For example, to create a new node configuration, you can run the command `v6 node new`, then you can start the new node by running `v6 node start`, and then stop the node with `v6 node stop` command.
+
+⚠️ Please make sure Docker is running when you're using the `v6 node` commands.
+
+
+## Configure a new node
+
+We will now create a new node configuration using the `v6 node new` command. 
+This process will create a configuration `yaml` file that the vantage6-node requires to run.
+
+Let's run the command:
+
+```bash
+v6 node new
+```
+
+The command will show a wizard to guide you through the configuration process in a step-by-step manner,
+
+```bash
+? Please enter a configuration-name: node1
+? Enter given api-key: ***
+? The base-URL of the server: https://portal.cotopaxi.vantage6.ai/
+? Enter port to which the server listens: 5000
+? Path of the api: /api
+? Task directory path: ***/vantage6/node/node1
+? Do you want to add a database? Yes
+? Enter unique label for the database: default
+? Database URI: ***/data.csv
+? Database type: csv
+? Do you want to add a database? Yes
+? Enter unique label for the database: default
+? Database URI: ***/data.sql
+? Database type: sql
+? Do you want to add a database? No
+? Which level of logging would you like? DEBUG
+? Do you want to connect to a VPN server? No
+? Enable encryption? No
+```
+
+1. The `configuration-name` is the name of the node configuration file. You can give any name you like.
+2. The `api-key` is the API key that you received from the vantage6 server administrator. It is used to authenticate the node at the server.
+3. The `base-URL of the server` is the URL of the vantage6 server. If you are running the server on your local machine using Docker, the URL has to be set to `http://host.docker.internal`.
+4. The `port to which the server listens` is the port number that the server listens to. It is usually `5000`.
+5. The `path of the api` is the path of the API of the server. It is usually `/api`.
+6. The `task directory path` is the path where the node will store the task files. You can give any path you like.
+7. The `database URI` is the path of the database file. You can add multiple databases by repeating the process. The database type can be 'csv', 'parquet', 'sql', 'sparql', 'excel' or 'omop'.
+8. The `level of logging` is the level of logging that you want. It can be 'DEBUG', 'INFO', 'WARNING', 'ERROR', or 'CRITICAL'.
+9. The `connect to a VPN server` is whether you want to connect to a VPN server. It is usually 'No', otherwise, you need to provide the VPN server URL.
+10. The `enable encryption` is whether you want to enable end-to-end encryption. If yes, you need to provide the encryption key.
+
+To see all configuration options, please check https://docs.vantage6.ai/en/main/node/configure.html#all-configuration-options.
+
+When you finish the configuration, you will see the following message,
```

Comment:
again I would use `:`

Reply:
which happens another few times below - I'm not repeating this comment every time